src/analyses/variable-sensitivity/abstract_value.h

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************\
+
+ Module: Analyses Variable Sensitivity
+
+ Author: Thomas Kiley, thomas.kiley@diffblue.com
+
+\*******************************************************************/
+#ifndef CPROVER_ANALYSES_VARIABLE_SENSITIVITY_ABSTRACT_VALUE_H
+#define CPROVER_ANALYSES_VARIABLE_SENSITIVITY_ABSTRACT_VALUE_H
+
+#include <analyses/variable-sensitivity/abstract_object.h>
+
+
+class abstract_valuet:public abstract_objectt
```

Comment:
Conceptually this is the idea of a primitive to mirror the array, pointer union and struct abstract objects. Me and @martin-cs discussed this, the conclusion to keep it was based on some of the transform resolution code would end up in here.